src/flock.rs

```diff
@@ -281,13 +276,15 @@ fn acquire(
 
         Err(e) => {
             if !error_contended(&e) {
-                return Err(e).wrap_err(format!("failed to lock file: {}", path.display()));
+                // .wrap_err(format!("failed to lock file: {}", path.display()))
+                Err(e)?;
             }
         }
     }
     eprintln!("Blocking: waiting for file lock on {}", msg);
 
-    lock_block().wrap_err_with(|| format!("failed to lock file: {}", path.display()))?;
+    // format!("failed to lock file: {}", path.display())
```

Comment:
It'd be really nice to keep having errors like this from flock, could we perhaps consider making this method return `FlockError` and adding some special cases for this?